django/contrib/postgres/serializer.py

```diff
@@ -0,0 +1,6 @@
+from django.db.migrations.serializer import BaseSerializer
+
+
+class DateTimeTZRangeSerializer(BaseSerializer):
+    def serialize(self):
+        return '%s' % repr(self.value), {}
```

Comment:
I am not sure why but I guess when i try to use a class from psycopg2.extras, I must import it. So that, if I check sys.path, I see that it has been registered.
For instance in django.tests.postgres_tests.test_apps.py, line 8
`from psycopg2.extras import DateTimeTZRange`